src/components/tables/TableItems.tsx

```diff
@@ -313,10 +313,16 @@ export const GroupedLinks = ({ data }: { data: ReviewTranscript }) => {
   const { pr_url } = data.review!;
   let publishUrl = "";
   const isPublished = data.review?.mergedAt;
-
+  const directoryUrlMatch = data.transcriptUrl
+    ? // checks for strings between contents/ and the last "/" (file name)
+      data.transcriptUrl.match(
+        /https:\/\/api\.github\.com\/repos\/[^\/]*\/[^\/]*\/contents\/((?:[^\/]*\/)+)/
+      )
+    : "";
+  const directoryPath = directoryUrlMatch ? directoryUrlMatch[1] : "";
   if (isPublished) {
-    let fileSlug = deriveFileSlug(data.content.title);
-    publishUrl = derivePublishUrl(fileSlug, data.content.loc);
+    let fileSlug = deriveFileSlug(data.content.title, /:/);
+    publishUrl = derivePublishUrl(fileSlug, directoryPath);
```

Comment:
The logic here should have a way to differentiate between the old implementation and the new one so we don't break the links of submissions that people had using the old implementation.

So the current way works for the old implementation, but not for the new one. For the new implementation you should use the `resolveGHApiUrl` in the same way that we are doing it in the `src/components/tables/TitleWithTags.tsx` file (you can actually see that logic as part of your committed changes in that file)